.github/scripts/build-examples.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -e
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source ${CURRENT_DIR}/common.sh
+
+# -- validate input ----------------------------------------------------------
+
+function help() {
+   echo
+   echo "Build examples from the repository"
+   echo
+   echo "Syntax: $0 fpga_family [examples]..."
+   echo "Arguments:"
+   echo "  fpga_family  - A supported FPGA family"
+   echo "  examples     - A name of an available example"
+   echo
+}
+
+if [[ ! $# -ge 1 ]]; then
+  echo "Invalid number of arguments!"
+  help
+  exit 1
+fi
+
+# -- tuttest -----------------------------------------------------------------
+
+fpga_family=$1
+shift
+
+examples="$@"
```

Comment:
Can you log a github issue to figure out a way to automatically determine these from the documentation? (Fixing not blocking merging.)

Reply:
https://github.com/SymbiFlow/symbiflow-examples/pull/88#discussion_r541060922